src/kotlin/org/intellij/plugins/hcl/formatter/HCLFormattingBuilderModel.kt

```diff
@@ -44,6 +44,7 @@ open class HCLFormattingBuilderModel(val language: Language = HCLLanguage) : For
         .after(EQUALS).spacing(spacesAroundAssignment, spacesAroundAssignment, 0, false, 0)
         .afterInside(IDENTIFIER, BLOCK).spaces(1)
         .afterInside(STRING_LITERAL, BLOCK).spaces(1)
+        .between(L_CURLY, R_CURLY).none()
```

Comment:
> Actually this change contradicts with one on next

It doesn't contradict with the next line, they work well together :) The easiest way to check it - to comment the next line and run tests. 

> and one four lines below

I'm not sure if it worked before within object because as far as I can see there were always a blank line there.